pkg/reconciler/v1alpha1/revision/resources/deploy.go

```diff
@@ -142,7 +142,7 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, observab
 			*userContainer,
 			*makeQueueContainer(rev, loggingConfig, autoscalerConfig, controllerConfig),
 		},
-		Volumes:                       []corev1.Volume{varLogVolume},
+		Volumes:                       append([]corev1.Volume{varLogVolume}, rev.Spec.Volumes...),
```

Comment:
In our runtime contract we have a set of default filesystems: https://github.com/knative/serving/blob/master/docs/runtime-contract.md#default-filesystems

Should we validate that the mount path for the volume is not interfere with the varLogVolume or other required paths?

Reply:
I think I'd want to answer https://github.com/knative/serving/pull/3061#issuecomment-459608218 first, which we may want to touch on the topic of collisions.

Reply:
I've disallowed `/tmp`, `/var/log`, and `/dev/log` specifically for now.  LMK if you think there are others I should add.

Reply:
Done